.buildkite/pipeline.yaml

```diff
@@ -25,28 +25,22 @@ steps:
 
   - label: "Release: test"
     if: "build.branch =~ /^wip_/"
+    plugins:
+      - ssh://git@github.com/sourcegraph/sg-buildkite-plugin.git#main: ~
     command: |
-      wget https://storage.googleapis.com/buildkite_public_assets/sg-rfc795.tar.gz
-      tar zxf sg-rfc795.tar.gz
-      chmod +x ./sg-rfc795
-
-      ./sg-rfc795 release run test --workdir=. --config-from-commit
+      sg release run test --workdir=. --config-from-commit
 
   - wait
 
   - label: "Release: finalize"
     if: "build.branch =~ /^wip_/"
+    plugins:
+      - ssh://git@github.com/sourcegraph/sg-buildkite-plugin.git#main: ~
```

Comment:
What is the `: ~` at the end?

Reply:
it means `NULL` basically, as in, this is an object, but there's nothing inside. This is usually to address the case where your yaml schema expects an object but all the fields are empty.